source/sixtrack.f90

```diff
@@ -929,6 +929,15 @@ subroutine daten
     end do
   end if
 
+  il1 = il
+  if(sixin_ncy2 == 0) il1 = il-1
```

Comment:
Consider adding a comment that this is (as far as I remember) due to skipping the last, automatically inserted, CAV "element".

Reply:
If you want me to add a comment, please provide the comment you want added.